dotCMS/dependencies.gradle

```diff
@@ -3,7 +3,7 @@ dependencies {
 
     def eeType = dotcmsReleaseVersion.equals("master") ? '-SNAPSHOT' : ''
 
-    compile group: 'com.dotcms.enterprise', name: 'ee', version: dotcmsReleaseVersion + eeType, changing: true
+    compile group: 'com.dotcms.enterprise', name: 'ee', version: 'issue-19646-write-directly-into-tar-gzip-SNAPSHOT', changing: true
```

Comment:
is this ok?

Reply:
No I have to remove this after the test run